mrmustard/lab_dev/circuit_components.py

```diff
@@ -183,27 +186,32 @@ def from_bargmann(
         wires = Wires(set(modes_out_bra), set(modes_in_bra), set(modes_out_ket), set(modes_in_ket))
         return cls._from_attributes(repr, wires, name)
 
-    @property
-    def bargmann(self) -> tuple:
+    def bargmann_triple(
+        self, batched: bool = False
+    ) -> tuple[Batch[ComplexMatrix], Batch[ComplexVector], Batch[ComplexTensor]]:
```

Comment:
I think `Batch`, `ComplexMatrix`, `ComplexVector` and `ComplexTensor` need to be imported based on codefactor